lib/ansible/plugins/connection/ssh.py

```diff
@@ -200,12 +200,12 @@
               which is why this feature is disabled by default.
           env:
             - name: ANSIBLE_PIPELINING
-            #- name: ANSIBLE_SSH_PIPELINING
+            - name: ANSIBLE_SSH_PIPELINING
           ini:
             - section: defaults
```

Comment:
@bcoca Is this going to be enabled in 2.10? It is not in 2.9

Thank you!

Reply:
we'll see, its grey area on the bugfix, will talk with release managers